tools/post-potd/post-potd.py

```diff
@@ -0,0 +1,37 @@
+from discord import SyncWebhook
+
+import json
+import requests
+
+from bs4 import BeautifulSoup
+
+
+def get_last_potd():
+    response = requests.get("https://leetcode.com/problemset/all/")
+    if response.status_code != 200:
+        raise "oops"
```

Comment:
Raise a more specific exception

Reply:
Link --> [useful exception handling for this case](https://www.geeksforgeeks.org/response-raise_for_status-python-requests/)

Reply:
How would it work with a try/except block?